packages/discord.js/src/structures/GuildMember.js

```diff
@@ -340,6 +345,11 @@ class GuildMember extends Base {
    * @param {?string} nick The nickname for the guild member, or `null` if you want to reset their nickname
    * @param {string} [reason] Reason for setting the nickname
    * @returns {Promise<GuildMember>}
+   * @example
+   * // Remove a nickname for a guild member
+   * guildMember.setNickname(null, "No nicknames allowed!")
+   *   .then(member => console.log(`Removed nickname for ${member.user.username}`))
+   *   .catch(console.log);
```

Comment:
I'm not against showing how to remove with `null`, but probably shouldn't be the only example.

Reply:
For consistency, the `catch()` here should call `console.error`.